Utils/hydro_utils.cpp

```diff
@@ -21,65 +21,105 @@ HydroUtils::ComputeFluxes ( Box const& bx,
                             Geometry const& geom, const int ncomp,
                             const bool fluxes_are_area_weighted )
 {
+#if (AMREX_SPACEDIM == 2)
+    if (geom.IsRZ()) {
+        // Need metrics when using RZ
+        Array<FArrayBox,AMREX_SPACEDIM> area;
+        Array<Elixir,AMREX_SPACEDIM> area_eli;
+        if (!fluxes_are_area_weighted) {
```

Comment:
I don't think the logic is right here. If we want area weighted fluxes, then we need to get the area FAB.